File: VpsApps/go/programs/internal/comics/monkeyuser.go
package comics

import (
	"internal/common"
	"strings"
)

// MonkeyUser Handle the Monkey User comic
func MonkeyUser() {
	url := "https://www.monkeyuser.com"
	data := FetchURL(url)
	if data == nil {
		return
	}

	text := string(data)
	pos := strings.Index(text, "div class=\"content\">")
	pos = IndexAt(text, "src", pos)
	startPos := pos + 5
	endPos := IndexAt(text, "\"", startPos)
	link := text[startPos:endPos]
	title := "Monkey User"
	common.HTMLHeader(title, url)
	common.HTMLAddComic(title, link, "900")
	common.HTMLLineBreak("")
}

File: various/azure-apps/AzureApps/Modules/Comics/Drivers/MonkeyUser.cs
using System.Text;
using dk.schau.AzureApps.Commands;
using dk.schau.AzureApps.Modules.Comics.DomainModel;

namespace dk.schau.AzureApps.Modules.Comics.Drivers
{
	public class MonkeyUser : IComicsDriver
	{
		readonly Fetcher _fetcher;

		public MonkeyUser(Fetcher fetcher)
		{
			_fetcher = fetcher;
		}

		public string Execute(string url, string name)
		{
			var response = _fetcher.ExecuteAsync(url).Result;
			var html = Encoding.UTF8.GetString(response);
			var pos = html.IndexOf("div class=\"content\">");
			pos = html.IndexOf("src", pos);
			var startPos = pos + 5;
			var endPos = html.IndexOf('"', startPos);
			var link = html.Substring(startPos, endPos - startPos);
			return $"<p><img src=\"{link}\" alt=\"{name}\" width=\"900\" /></p><p><br /></p>";
		}
	}
}